AppVeyor.psm1

```diff
@@ -53,13 +53,26 @@ function Invoke-AppveyorInstallTask
                               -ChildPath 'nuget.exe'
     Install-NugetExe -OutFile $nugetExePath
 
+    $pesterParameters = @{
+        Name = 'Pester'
+        Force = $true
+    }
+
+    $installModuleSupportsSkipPublisherCheck = (Get-Command Install-Module).Parameters['SkipPublisherCheck']
+    if ($installModuleSupportsSkipPublisherCheck)
+    {
+        $pesterParameters += @{
+            SkipPublisherCheck = $true
+        }
+    }
+
     if ($PesterMaximumVersion)
     {
-        Install-Module -Name Pester -MaximumVersion $PesterMaximumVersion -Force -SkipPublisherCheck
+        Install-Module @pesterParameters -MaximumVersion $PesterMaximumVersion
```

Comment:
Now that the parameters for this call are separated out, the MaximumVersion parameters should be separated out too. Like this:
```powershell
if ($PesterMaximumVersion)
{
    $pesterParameters['MaximumVersion'] = $PesterMaximumVersion
}

Install-Module @pesterParameters
```
That way there is only ever one call to Install-Module and any parameter differences happen within the same hashtable variable.

Also the variable pesterParameters should be renamed intallPesterParameters to make it clear that they are parameters for installing Pester and not invoking Pester or doing something else with it.